docs/docs.scss

```diff
@@ -1,22 +1,104 @@
 @import "../styles/style-guide.scss";
 @import "../bower_components/pygments/css/default.css";
 
-.swatch {
+$swatch-padding: 5px;
+$swatch-width-height: 90px;
+$swatch-label-height: 30px;
+
+.swatch-with-label {
     display:inline-block;
-    width: 6rem;
-    height: 6rem;
-    border-radius: 0.25rem;
-    border-color: black;
+    width: $swatch-width-height;
+    height: $swatch-width-height + $swatch-label-height + 2px;
+    vertical-align: top;
+    margin: 4px;
+
+    label{
+        background-color:white;
+        width:$swatch-width-height;
+        display:block;
+        height: $swatch-label-height;
+        font-size: 1.2em;
+        margin: 4px;
+        text-align: center;
+        font-weight: bold;
+    }
+    .swatch {
+        display:block;
```

Comment:
Properties should be ordered border-radius, box-shadow, display, height, padding, vertical-align, width<br>Colon after property should be followed by one space